model/src/application/interaction/application_command/option.rs

```diff
@@ -326,6 +351,7 @@ pub enum CommandOptionValue {
     SubCommand(Vec<CommandDataOption>),
     SubCommandGroup(Vec<CommandDataOption>),
     User(Id<UserMarker>),
+    Focused(String, CommandOptionType),
```

Comment:
Keep alphabetical

Reply:
Also might be nice to document why this variant has its own `CommandOptionType`

Reply:
I've added a comment, is it how you've imagined?